<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>work_complete</class>
 <widget class="QDialog" name="work_complete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#work_complete{
	
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(235, 153, 21, 249), stop:0.81592 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 117, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Cambria&quot;;
font:bold;
color: white;
</string>
    </property>
    <property name="text">
     <string>Finished Orders</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="from_cal_tool_btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>30</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/calendar-512.png</normaloff>images/calendar-512.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="from_date">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>161</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="from_label">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Cambria&quot;;
font:bold;
color: rgb(170, 0, 0);
</string>
   </property>
   <property name="text">
    <string>From :</string>
   </property>
  </widget>
  <widget class="QToolButton" name="to_cal_tool_btn">
   <property name="geometry">
    <rect>
     <x>1121</x>
     <y>160</y>
     <width>30</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/calendar-512.png</normaloff>images/calendar-512.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="to_date">
   <property name="geometry">
    <rect>
     <x>931</x>
     <y>161</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="to_label">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Cambria&quot;;
font:bold;
color: rgb(170, 0, 0);
</string>
   </property>
   <property name="text">
    <string>To  :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(37, 125, 40, 255), stop:1 rgba(255, 255, 255, 255));
font: 75 14pt &quot;Calibri&quot;;
border-radius:25px;
border-style:outset;
border-width:5px;
border-color:green;
font:bold;</string>
   </property>
   <property name="text">
    <string>Show All Finished</string>
   </property>
  </widget>
  <widget class="QFrame" name="finish_frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>1221</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#finish_frame{border-width:1.5px;
border-style:outset;
border-color:black;
border-width:4px;}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="finish_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1181</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#finish_table{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 255, 255, 255));}</string>
    </property>
    <column>
     <property name="text">
      <string>Bill No</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Customer Name</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone No</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Event Name</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Booking Date</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total Amount</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Date</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Finish Date</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Studio Name</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>751</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  
    border: 3px solid brown;
    border-radius: 15px;
    margin-top: 6px;
	color:black;
	font: 75 13pt &quot;Calibri&quot;;
 	font: bold;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(64, 136, 217, 255), stop:1 rgba(255, 255, 255, 255));
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position :top center;
    padding: 0px 0px 0px 0px;
}
</string>
   </property>
   <property name="title">
    <string>Select Date Range</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>frame</zorder>
  <zorder>from_cal_tool_btn</zorder>
  <zorder>from_date</zorder>
  <zorder>from_label</zorder>
  <zorder>to_cal_tool_btn</zorder>
  <zorder>to_date</zorder>
  <zorder>to_label</zorder>
  <zorder>finish_btn</zorder>
  <zorder>finish_frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
